src/router/index.js

```diff
@@ -15,7 +15,13 @@ const routes = [
     path: '/',
     name: 'Home',
     component: () => import('../pages/HomeSelector.vue'),
-    meta: { pageTitle: 'Dashboard' },
+    meta: {
+      pageTitle: {
+        'en-US': 'Dashboard',
+        en: 'Dashboard',
+        es: 'Tablero',
+      },
+    },
```

Comment:
This is the only file so far that does not use translations from the external files in `translations/*/*.json`. Could we change the router files to also use translations from that directory? I'm just thinking that it would be nice if all of the translation content was in that folder.

Reply:
I am looking for a better solution to hard coding this way.

However, I wrote it this way because the i81n instance is only defined with a <script setup> function, so we are unable to directly insert translations into the router file since it lacks a setup function.

Reply:
Could we do something like
```
import pageTitles from `../translations/pageTitles.json` assert { type: `json` };
```

and then reference the `pageTitles` object in the code.

Reply:
Or perhaps they should be located in the individual language sub-folders.